ui/app/models/agent.js

```diff
@@ -29,9 +29,12 @@ export default class Agent extends Model {
     return formatHost(address, rpcPort);
   }
 
-  @computed('rpcAddr', 'system.leader.rpcAddr')
-  get isLeader() {
-    return this.get('system.leader.rpcAddr') === this.rpcAddr;
+  @tracked isLeader = false;
+
+  @action async checkForLeadership() {
+    const leaders = await this.system.leaders;
+    this.isLeader = leaders.includes(this.rpcAddr);
+    return this.isLeader;
```

Comment:
using `tracked` is more idiomatic than the isLeader getter in Ember Octane. And since service getters (like `this.get('system.leader....)` was the thing holding it back as a classic model, removed the `@classic` tag as well.